common/models/types/index.d.ts

```diff
@@ -78,6 +109,13 @@ declare interface LexicalModel {
    * @see LexicalModelPunctuation
    */
   readonly punctuation?: LexicalModelPunctuation;
+
+  /**
+   * Lexical models _may_ provide a LexiconTraversal object usable to enhance 
+   * prediction and correction results.  The returned object represents the
+   * unfiltered lexicon (with an empty prefix).
+   */
+  getRootTraversal?(): LexiconTraversal;
```

Comment:
Using my suggestion from before...

```suggestion
  traverseFromRoot?(): LexiconTraversal;
```